src/screens/MapScreen.tsx

```diff
@@ -1,9 +1,9 @@
-import { Text, View, StyleSheet } from "react-native";
+import { View, StyleSheet } from "react-native";
+// import UserLocation from "../UserLocation";
+import LTest from "../location";
 
 export const MapScreen = () => (
-  <View style={styles.container}>
-    <Text style={styles.huge}>Map Screen</Text>
-  </View>
+  <View style={styles.container}>{<LTest />}</View>
```

Comment:
Okay I get this is for testing at the moment, but this pattern is an eyesore for production. Later on, it would be best if the location code was a hook that we could call where we want. In theory there shouldn't be a component attached to the location fetching code I think, right?

Reply:
Nah it's fine for now, just some ideas for when you're closer to the final version.

Reply:
This file wont look like this for final version anyways